MultimodalQnA/tests/test_compose_on_gaudi.sh

```diff
@@ -56,10 +57,11 @@ function build_docker_images() {
     fi
 
     cd $WORKPATH/docker_image_build
-    git clone --depth 1 --branch ${opea_branch} https://github.com/opea-project/GenAIComps.git
+    #git clone --depth 1 --branch ${opea_branch} https://github.com/opea-project/GenAIComps.git
+    git clone --depth 1 --single-branch --branch="mmqna-phase3" https://github.com/mhbuehler/GenAIComps.git
```

Comment:
We have the test files cloning our coordinating `mmqna-phase3` GenAIComps branch. Once our GenAIComps PR has been merged, we can revert these changes.

Reply:
Our corresponding GenAIComps PR has been merged, so this has been reverted.